<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgPresetSelectorStrip</class>
 <widget class="QWidget" name="WdgPresetSelectorStrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>50</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="leftScrollBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>65</number>
     </property>
     <property name="autoRepeatInterval">
      <number>65</number>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KisPresetChooser" name="smallPresetChooser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="rightScrollBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>65</number>
     </property>
     <property name="autoRepeatInterval">
      <number>65</number>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisPresetChooser</class>
   <extends>QWidget</extends>
   <header>widgets/kis_preset_chooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>